app/services/catalog/service_plan_json.rb

```diff
@@ -21,7 +21,7 @@ def process
       relevant_service_plans.each do |plan|
         @reference = plan.portfolio_item.service_offering_ref
         @portfolio_item_id = plan.portfolio_item.id
-        @modified = plan.modified.present?
+        @modified = true
```

Comment:
Per our conversation on this. Since we now set it `true`after import of a service plan. I would like to see the instance variable renamed to `@imported` and the flag sent back in the payload renamed as well.

Wouldn't mind your thoughts on this @Hyperkid123

Reply:
I have no issues with a different name of the flag.

Reply:
Updated to use `imported` instead of `modified` both in the response and in the code.